crypto/src/main/java/org/cryptimeleon/incentive/crypto/IncentiveSystem.java

```diff
@@ -873,28 +907,3 @@ private SpendDeductRandomness computeSpendDeductRandomness(UserSecretKey userSec
     }
 }
 
-/**
```

Comment:
why removed?

Reply:
Moved them to separate files as it is better practise